tests/test_explorer.py

```diff
@@ -29,19 +35,35 @@ def test_init_thread(self):
             self.assertEqual(thread.widget, widget)
             mockedFetched.connect.assert_called_once_with(callback, type=Qt.QueuedConnection)
 
-    @mock.patch("iquip.apps.explorer.cmdtools.run_command")
-    def test_run(self, mocked_run_command):
-        mocked_run_command.return_value.stdout = "path1\npath2\n"
+    def test_run(self):
+        self.mocked_response.json.return_value = ["path1", "path2"]
         widget = QTreeWidgetItem()
         parent = QObject()
         with mock.patch("iquip.apps.explorer._FileFinderThread.fetched") as mockedFetched:
             thread = explorer._FileFinderThread(path="path", widget=widget,
                                                 callback=mock.MagicMock(), parent=parent)
             thread.run()
             thread.wait()
-            mocked_run_command.assert_called_once_with("artiq_client ls path")
+            self.mocked_get.assert_called_once_with("http://127.0.0.1:8000/ls/",
+                                               params={"directory": "path"},
+                                               timeout=10)
```

Comment:
Please follow one of the following styles:
```python
object.method(arg1,
              arg2,
              arg3)
```
or
```python
object.method(
    arg1,
    arg2,
    arg3,
)
```
or
```python
object.method(
    arg1, arg2, arg3)
```

Reply:
Oh, sorry😅 I intended the first suggestion, but it seems to have been pushed back